Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings

```diff
@@ -226,6 +238,18 @@
 /* Specific route feedback that route contained impassible, narrow roads. */
 "ROUTE_QUALITY_ROADS_TOO_NARROW_FEEDBACK" = "Route had roads too narrow to pass";
 
+/* Alternatives selection note about equal travel distance. */
+"SAME_DISTANCE" = "Same distance";
```

Comment:
Let's create a follow-up ticket to localize these newly added strings.

Reply:
Added https://github.com/mapbox/mapbox-navigation-ios/issues/3964